# ==============================================================================
# FIXING COMMIT DIFF
# ==============================================================================
# File: testing/web-platform/tests/selection/crashtests/selection-detached-range-find-crash.html
# Commit: c51bc6c076a9
# Full Hash: c51bc6c076a9d418078ab86c919f56420a30d339
# Author: Jan-Niklas Jaeschke <jjaschke@mozilla.com>
# Date: 2025-03-07 21:43:55
# Description:
#   Bug 1949877 - nsFind: Return null if boundary points are removed from the DOM tree. r=emilio
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D240494
# ==============================================================================

diff -r bdf0a26f9ce0 -r c51bc6c076a9 testing/web-platform/tests/selection/crashtests/selection-detached-range-find-crash.html
--- /dev/null	Thu Jan 01 00:00:00 1970 +0000
+++ b/testing/web-platform/tests/selection/crashtests/selection-detached-range-find-crash.html	Fri Mar 07 09:09:24 2025 +0000
@@ -0,0 +1,21 @@
+<script>
+    document.addEventListener("DOMContentLoaded", () => {
+      let s = window.getSelection()
+      r = document.createRange()
+      r.selectNode(c)
+      s.addRange(r)
+      r.selectNode(a)
+      s.addRange(r)
+      window.getSelection().modify("move", "right", "line")
+      b.replaceWith(document.createElement("section"))
+      window.find("A")
+    })
+    </script>
+    <del>
+    <strong id="a"></strong>
+    <menu>
+    <canvas>
+    <input/>
+    <mark>
+    <small id="b">
+    <video id="c">